app/views/about/_login.html.haml

```diff
@@ -19,4 +19,4 @@
 
     .actions
       - User.omniauth_providers.each do |provider|
-        = provider_sign_in_link(provider)
+        = provider_sign_in_link(provider)
```

Comment:
Extra newline change! This would be cleaner if you just ignored changes to that file.